### DISCUSSION TITLE
Edit Assembly process Name

### DISCUSSION THREAD

**Discussion by beastdrc** [dnSpyEx/dnSpy#147] [Q&A]
Hey, Guys. I'm trying some reverse. I want to analyze app traffic using Fiddler. But the problem is when I start target program it detects fiddler and
1) Closes Fiddler
2) Show message, that it cant work with fiddler running

I'm trying to google solution, with no success 
I changed some properties in AssemblyAttributes, but with no success
![image](https://user-images.githubusercontent.com/38570368/204238814-f57c97b9-c5fb-443b-bb6a-08f9088f0d71.png)

![image](https://user-images.githubusercontent.com/38570368/204238873-6f9d6eb7-8906-41db-bcf3-88b2f25b4a64.png)

**Comment by GazziFX** [comment]
Not sure its question for dnSpy, but you can try to edit Version Info with Resource Hacker
![reshacker](https://user-images.githubusercontent.com/43498426/204273729-e7327f6c-820a-4acb-ad9c-1e3ae207c11b.jpg)

**Reply by beastdrc** [reply]
Thanks! this worked for me name changed. will try to test if app will be able discover fiddler. 
P.S Ты мне не первый раз помогаешь, спасибо!

**Comment by ElektroKill** [comment]
Hello,

This discussion board is not meant for asking for help with general reverse engineering related questions. This discussion board is for discussing the dnSpyEx project specifically. Think of things like the current/future features of dnSpyEx, suggestions, bugs, questions relating to dnSpyEx, etc. I will be locking this discussion now!

Furthermore, next time please speak in English so it is more understandable!

Thanks!
